CHANGELOG.md

```diff
@@ -5,12 +5,32 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 since 0.4.1, and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## [Unreleased](https://github.com/freedomofpress/dangerzone/compare/v0.7.1...HEAD)
+## [0.8.0](https://github.com/freedomofpress/dangerzone/compare/v0.8.0...0.7.1)
 
 ### Added
 
 - Point to the installation instructions that the Tails team maintains for Dangerzone ([announcement](https://tails.net/news/dangerzone/index.en.html))
-- Platform support: Ubuntu 24.10 and Fedora 41 ([issue #947](https://github.com/freedomofpress/dangerzone/issues/947))
+- Installation and execution errors are now catched and displayed in the interface ([#952](https://github.com/freedomofpress/dangerzone/pull/952))
+- Prevent users from using illegal characters in output filename ([#942](https://github.com/freedomofpress/dangerzone/pull/942)). Thanks @bnewc for the contribution!
+- Add support for Fedora 41 ([#953](https://github.com/freedomofpress/dangerzone/pull/953))
+- Add support for Ubuntu 24.10 ([#954](https://github.com/freedomofpress/dangerzone/pull/954))
+
+### Fixed
+
+- Do not close stderr to read the debug logs on Qubes ([#967](https://github.com/freedomofpress/dangerzone/pull/967))
+- Update our macOS entitlements, removing now unneeded privileges ([#639](https://github.com/freedomofpress/dangerzone/pull/639))
+
+### Changed
+
+- The second phase of the conversion (pixels to PDF) now happens on the host. Instead of first grabbing all of the pixel data from the first container, storing them on disk, and then reconstructing the PDF on a second container, Dangerzone now immediately reconstructs the PDF **on the host**, while the doc to pixels conversion is still running on the first container. The sanitzation is no less safe, since the boundaries between the sandbox and the host are still respected ([#748](https://github.com/freedomofpress/dangerzone/pull/748))
+- Sign Windows executables and installer with SHA256 rather than SHA1 ([#931](https://github.com/freedomofpress/dangerzone/pull/931)). Thanks @jkarasti for the contribution!
+- PyMuPDF is now vendorized for Debian packages ([#940](https://github.com/freedomofpress/dangerzone/pull/940))
```

Comment:
I'd maybe add here: "This is done because the PyMuPDF package from the Debian repos lacks OCR support."

Reply:
Why not! Generally speaking, I believe it's best to keep these messages short if possible, and people can click the links to get more information. There are a few cases where I decided this to not be the case (for instance the on-host conversion).

I've added the context you're pointing anyway, but let's discuss on the general topic to see if we agree, maybe?

Reply:
(done in 178a1f705ff1a2a4908027543f81119df0cd1214)